L/libjulia/libjulia@1.7/bundled/patches/0003-exports.patch

```diff
@@ -0,0 +1,19 @@
+diff --git a/src/jl_exported_funcs.inc b/src/jl_exported_funcs.inc
+index 82611bfb50..81e68f3b78 100644
+--- a/src/jl_exported_funcs.inc
++++ b/src/jl_exported_funcs.inc
+@@ -149,6 +149,7 @@
+     XX(jl_expand_with_loc) \
+     XX(jl_expand_with_loc_warn) \
+     XX(jl_extern_c) \
++    XX(jl_field_index) \
+     XX(jl_gc_add_finalizer) \
+     XX(jl_gc_add_finalizer_th) \
+     XX(jl_gc_add_ptr_finalizer) \
+@@ -540,4 +541,5 @@
+     XX(jl_vprintf) \
+     XX(jl_wakeup_thread) \
+     XX(jl_yield) \
+-    XX(jl_print_backtrace)
++    XX(jl_print_backtrace) \
++    XX(jl_get_pgcstack)
```

Comment:
The other day I was confused because I thought you already applied this patch

Reply:
bad timeline of events: I wasn't in a position to update my other libjulia PR to add this in, and only now got to it.

Hmm, one job failed:
> ERROR: LoadError: GitError(Code:ELOCKED, Class:Index, the index is locked; this might be due to a concurrent or crashed process)

I guess that's issue #3242. Gotta restart that job?